featurflag.ps1

```diff
@@ -0,0 +1,60 @@
+[CmdletBinding(PositionalBinding=$false)]
+param(
+  [string]$flag = $null,
+  [switch]$enable,
+  [switch]$disable,
+  [string]$hive = "RoslynDev",
+  [switch]$set,
+  [switch]$get
+)
+
+if ($null -eq $flag -or '' -eq $flag) {
+    throw "Specify a -flag to set"
+}
+
+$flagBase = "FeatureFlags\Razor\LSP"
+$slashIndex = $flag.LastIndexOf("\")
+
+if ($slashIndex -ge 0) {
+    $flagBase = $flag.Substring(0, $slashIndex)
+    $flag = $flag.Substring($slashIndex + 1) #+1 to trim the \
```

Comment:
nit: seems like this would be out of bounds if slash was the right char of $flag :)

Reply:
I added a check for ending with "\" since that wouldn't be valid